src/components/HomePage/UpcomingEvents.js

```diff
@@ -10,8 +10,9 @@ import workshopSchedule from '../../data/WorkshopSchedule';
 import Event from './Event';
 import NoSsr from '@material-ui/core/NoSsr';
 
-// When changing this, make sure to update src/components/SchedulePage/Schedule.js as well.
-const scheduleAvailable = false;
+// Schedule will be available on the homepage the day before HOTH
+const scheduleAvailableTime = new Date('2022-02-25T00:00:00-08:00');
+const scheduleAvailable = Date.now() > scheduleAvailableTime.getTime();
```

Comment:
As long as you've made sure to test this when `scheduleAvailableTime` is not in the future but in the past, and make sure that they have the correct behavior, then this looks fine to me.

Reply:
Yup! I set it to earlier today and the schedule showed up :)